stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json

```diff
@@ -138,5 +141,27 @@
             "subject": "ssl_subject",
             "subject_public_key_algorithm": "ssl_publickey_algorithm"
         }
+    },
+    "x-splunk": {
+        "fields": {
+            "log_source": ["source"],
+            "provider": ["_sourcetype"],
+            "device_type": ["DeviceType"],
+            "direction": ["Direction"],
+            "severity": ["severity"],
+            "EventID": ["EventID"],
+            "event_name": ["EventName"],
+            "Mitre_TacticId": ["TacticId"],
+            "Mitre_Tactic": ["Tactic"],
+            "Mitre_TechniqueId": ["TechniqueId"],
+            "Mitre_Technique": ["Technique"]
+        }
+    },
+    "x-ibm-finding": {
+        "cim_type": "endpoint",
+        "fields": {
+          "ss_name": ["ss_name"],
+          "severity": ["severity"]
+        }
     }
-}
\ No newline at end of file
+}
```

Comment:
I think you are not supposed to make changes in `cim_from_stix_map.json`. we need to have a `from_stix_map.json` file inside `stix_shifter_modules/splunk/stix_translation/json`. Make a copy of `cim_from_stix_map.json` and put it inside splunk module and make changes there.

Reply:
@mdazam1942 
Okay,Noted. We have resolved unittest. we'll move mapping files in splunk module and will push changes

Reply:
Moved mapping files in splunk module, please review